File: src/test/java/TC/삼총사Test.java
package TC;

import org.assertj.core.api.Assertions;
import org.example.TC.삼총사;
import org.junit.jupiter.api.Test;

public class 삼총사Test {
    private final 삼총사 three = new 삼총사();

    @Test
    void solution(){
        int[] number = {-2, 3, 0, 2,-5};
        int answer = 2;
        int result = three.solution(number);
        Assertions.assertThat(result).isEqualTo(answer);
    }
    @Test
    void solution2() {
        int[] number = {-3,-2,-1,0,1,2,3};
        int answer = 5;
        int result = three.solution(number);
        Assertions.assertThat(answer).isEqualTo(result);
    }
}

File: src/test/java/TC(K)/삼총사Test.kt
package `TC(K)`

import io.kotest.core.spec.style.StringSpec
import nonapi.io.github.classgraph.utils.Assert
import org.assertj.core.api.Assertions
import org.example.TCK.삼총사

class 삼총사Test:StringSpec(
    {
        "삼총사 테스트 1"{
            val number:Array<Int> = arrayOf(-2, 3, 0, 2, -5)
            val answer:Int = 2
            val result:Int = 삼총사().solution(number)
            Assertions.assertThat(result).isEqualTo(answer)
        }

        "삼총사 테스트 2" {
            val number:Array<Int> = arrayOf(-3, -2, -1, 0, 1, 2, 3)
            val answer = 5
            var result = 삼총사().solution(number)
            Assertions.assertThat(answer).isEqualTo(result)
        }
    }
)
